CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 3.4.2 - 2022-07-11
```

Comment:
if we get quick testing results I would do the release even tomorrow. we can leave this unchanged until we know the date :+1: